lms/static/styles/components/_FilePickerApp.scss

```diff
@@ -32,8 +29,9 @@
 }
 
 .FilePickerApp__document-source-buttons {
```

Comment:
I could have also added the vertical rhythm mixin here, but since it was unchanged from the util class I opted to include the actual exported class in the component directly. "u-vertical-rhythm"

Reply:
Danger zone! Recall from this PR yesterday: https://github.com/hypothesis/frontend-shared/pull/66 that all utility classes are about to get a prefix. Also, there is documentation here: https://github.com/hypothesis/frontend-shared/tree/main/styles about what the current package SASS API is, and (in the FAQ) section, specifically advises against using any utility classes directly. In short, this class won't exist after the next package release.

Reply:
See comment further down: this will break on the next `frontend-shared` package release.